earth_enterprise/src/fusion/portableglobe/servers/local_server.py

```diff
@@ -409,9 +410,19 @@ def ParseGlobeReqName(self, req):
     else:
       # Globe requested is the current selectedGlobe
       return 1
-    
 
-  def LocalJsonHandler(self, handler, is_2d=False):
+  def JStoJson(self, js_string):
+    """Converts a JS server definition string to valid JSON."""
+    # Remove "var geeServerDefs = " or similar from start.
+    # Then add quotes to JSON keys that don't have them.
+    # Strip out the trailing ';'
+    # Finally, push it through json.dumps to ensure consistently-formatted output.
+    out_string = re.sub(r"^var \w+ ?= ?", "", js_string)
```

Comment:
This is fairly minor, but is this code duplicated in the regular server code? If it's not too much work it would be nice to have them shared.

Reply:
I considered this!  But after looking at where it's located in the codebase, it feels like trying to access it here (or restructuring the data flow so it can be converted someplace where that code is easily accessed) would add more complication and risk of problems than it would avoid.

Reply:
I'm good with that.